superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -329,7 +391,7 @@ int DestroyBuf(const BenchArgs &args, Buffers *buffers) {
                 fprintf(stderr, "DestroyBuf::cudaHostUnregister error: %d\n", cuda_err);
                 ret = -1;
             }
-            numa_free(*(host_buf_ptrs[i]), args.size);
+            numa_free(*(host_buf_ptrs[i]), args.is_bidirectional ? args.size * 2 : args.size);
```

Comment:
Why not args.size * 2 directly?

Reply:
Say for unidirectional test for message size S, we are creating a buffer pair containing 1 size S buffer on device A and 1 size S on device B. Then for bidirectional test, we need to create another such buffer pair. `args.size * 2` here is for implementation of such buffer pair in bidirectional case.

Reply:
Now each sub-benchmark has their own buffers. No *2 hack anymore.